security/package-requirements.md

```diff
@@ -0,0 +1,16 @@
+## SSWG security requirements for packages on the index
+
+This information is intended for package authors who have packages listed on the SSWG's package index or are looking into pitching/proposing their package to be listed. If instead you found (or have heard of) a security vulnerability you’d like to report, please have a look over [here](contributor-found-vulnerability.md).
+
+---
+
+Packages that are listed on the [SSWG's package index](https://swift.org/server/#projects) are required to follow the following guidelines around security.
+
+Where security vulnerabilities are involved, it is key to ensure that somebody who discovers a vulnerability in your package can immediately find information on how to report it. As the package author, you know best where to put important information about your software. In general, it is typically the `README.md` file in the root of your repository on GitHub/GitLab/etc. The phrase "main information page" is used elsewhere in this document to refer to the `README.md` or other resource as appropriate for your project.
```

Comment:
Might be worth explicitly calling out the "special cased" security pages. e.g. on GitHub "SECURITY.md" (probably other extensions too) will be automagically listed as the security policy for the project (like: https://github.com/apple/swift-nio/security/policy) which is also linked from the new issue page.

Reply:
@swift-server/sswg / @glbrntt  Thanks George, that's a great idea. I reworded it and now recommend `SECURITY.md` as the recommended place for the security policy (and point out that Github has special support).

WDYT?